apps/reference/docs/guides/cli/environments.mdx

```diff
@@ -146,30 +144,38 @@ GRANT ALL ON TABLE public.employees TO postgres;
 GRANT ALL ON TABLE public.employees TO service_role;
 ```
 
-You may notice that the auto generated migration script is more verbose than the manually written one. This is because the schema diff tool we use under the hood does not take into account of default privileges added by the initial schema.
+You may notice that the auto generated migration script is more verbose than the manually written one.
+This is because the default schema diff tool does not take into account of default privileges added by the initial schema.
 
-Alternatively, you may pass in the `--use-migra` experimental flag to generate a more concise schema diff.
+Finally, commit the new migration script to git and you are ready to deploy.
 
-```sql
-supabase db diff --use-migra
-```
+:::tip
 
+Alternatively, you may pass in the `--use-migra` experimental flag to generate a more concise migration using [migra](https://github.com/djrobstep/migra).
 Without the `-f` file flag, the output will be written to stdout by default.
 
-Finally, commit the new migration script to git and you are ready to deploy 🎉
+`supabase db diff --use-migra`
+
+:::
 
 ## Deploying a migration
 
-In a real project, you might not want to deploy migrations directly from your local machine as other engineers could be pushing features simultaneously. Instead, you would use a CI/CD pipeline to deploy new migrations.
+In a production environment, we recommend use a CI/CD pipeline to deploy new migrations, rather than deploying from your local machine. We can achieve this by using GitHub Actions.
```

Comment:
```suggestion
In a production environment, we recommend using a CI/CD pipeline to deploy new migrations with GitHub Actions rather than deploying from your local machine.
```